astronomer/providers/google/cloud/example_dags/example_gcp_nuke.py

```diff
@@ -0,0 +1,79 @@
+import os
+from datetime import timedelta
+
+from airflow import DAG
+from airflow.models.variable import Variable
+from airflow.operators.python import PythonOperator
+from airflow.utils.timezone import datetime
+
+REGION = os.getenv("GCP_LOCATION", "us-central1")
+EXECUTION_TIMEOUT = int(os.getenv("EXECUTION_TIMEOUT", 6))
+
+default_args = {
+    "execution_timeout": timedelta(hours=EXECUTION_TIMEOUT),
+    "retries": int(os.getenv("DEFAULT_TASK_RETRIES", 2)),
+    "retry_delay": timedelta(seconds=int(os.getenv("DEFAULT_RETRY_DELAY_SECONDS", 60))),
+}
+
+
+def gcloud_nuke_callable() -> None:
+    """Deletes stale GCP resources."""
+    import json
+    import logging
+    import subprocess
+    import tempfile
+
+    sa_json = Variable.get("gcp_nuke_sa_json", deserialize_json=True)
+
+    with tempfile.NamedTemporaryFile("w+") as temp_file:
+        json.dump(sa_json, temp_file)
+        temp_file.flush()
+        sa_file_name = temp_file.name
+
+        # Authenticate with GCP using service account file.
+        subprocess.getoutput(f"gcloud auth activate-service-account --key-file={sa_file_name}")
+        subprocess.getoutput("gcloud config set project astronomer-airflow-providers")
```

Comment:
Can we get the project name from environment variables than hardcoding to `astronomer-airflow-providers`